src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__college_assessment_dashboard.sql

```diff
@@ -66,7 +66,39 @@ with
             )
     ),
 
-    act_sat_official as (
+    course_subjects_roster as (
+        select
+            e._dbt_source_relation,
+            e.academic_year,
+            e.student_number,
+
+            s.courses_course_name,
+            s.teacher_lastfirst,
+            s.sections_external_expression,
+            s.courses_credittype,
+
+            adb.contact_id,
+
+        from {{ ref("base_powerschool__student_enrollments") }} as e
+        left join
+            {{ ref("base_powerschool__course_enrollments") }} as s
+            on e.studentid = s.cc_studentid
+            and e.academic_year = s.cc_academic_year
+            and {{ union_dataset_join_clause(left_alias="e", right_alias="s") }}
+        left join
+            {{ ref("int_kippadb__roster") }} as adb
+            on e.student_number = adb.student_number
+        where
+            e.rn_year = 1
+            and e.school_level = 'HS'
+            and e.schoolid != 999999
+            and s.rn_course_number_year = 1
+            and not s.is_dropped_section
+            and s.courses_credittype in ('ENG', 'MATH')

```

Comment:
move these to the join

Reply:
oohh, interesting. i want to learn more about why. do you have a preferred source i can read on when it is more efficient to use these on joins rather than on where clauses?

Reply:
I don't have a go-to source but it's just better organized imo

Reply:
LMAO! fixed!